warcio/cli.py

```diff
@@ -94,6 +100,12 @@ def indexer(cmd):
     indexer.process_all()
 
 
+# ============================================================================
+def checker(cmd):
+    checker = Checker(cmd)
```

Comment:
This is more of nit but I would prefer it if variables did not shadow the function name from the outer scope. 
@ikreymer thoughts?

Reply:
Yeah, would be better, can just use `_checker`. Should do the same for `_indexer` in indexer()